qa/rpc-tests/mn_tickets.py

```diff
@@ -512,6 +518,76 @@ def personal_royalty_initialize_tests(self):
         print("Personal royalty initialize tested")
 
     # ===============================================================================================================
+    def generate_art_ticket_details(self):
+        # Art_ticket structure 
+        # {
+        # "artist_name": string,
+        # "artwork_title": string,
+        # "artwork_series_name": string,
+        # "artwork_keyword_set": string,
+        # "artist_website": string,
+        # "artist_written_statement": string,
+        # "artwork_creation_video_youtube_url": string,
+        # "thumbnail_hash": bytes,    //hash of the thumbnail !!!!SHA3-256!!!!
+        #     "data_hash": bytes,         // hash of the image (or any other asset) that this ticket represents !!!!SHA3-256!!!!
```

Comment:
not accurate

Reply:
Could you please specify ? I got this from ayk so please correct me here !

Reply:
it is not bug, just place text accurate on the same level with other lines
"data_hash": bytes,         // hash of the image (or any other asset) that this ticket represents !!!!SHA3-256!!!!